### ISSUE TITLE
Attempt to clone a vm with "clonevm" command produces confusing error

### ISSUE THREAD

**Issue by gurevichmark** [xcat2/xcat-core#2805]
Running a `clonevm` command following an example in man page, produces a confusing message:

```
[root@fs2vm102 ~]# clonevm lsvm_test1 -t appserver
lsvm_test1: Error: //<server>/<path>/<mastername> support for cloning at this moment
[root@fs2vm102 ~]#
```

Also, reading the code, it seems that `clonevm` is only supported for VM with NFS mounted disks. Should documentation pages be modified to make that clear to the user ?

**Comment by whowutwut** [comment]
Not in backlog, removing sprint label

**Comment by gurevichmark** [comment]
Fixed by PR #2965. The new output is now:

```
[root@c910f02c05p03]# clonevm fs2vm105 -t appserver
fs2vm105: Error: VM cloning is only supported for nfs server vmstorage attribute.

[root@c910f02c05p03]# echo $?
1
```
